### ISSUE TITLE
Bug: Plugin losing all Job info when taking the Calais - Folkestone Train.

### ISSUE THREAD

**Issue by Pnaop** [RenCloud/scs-sdk-plugin#90]
**Describe the bug**
When using the plugin, all the job data is lost (screenshot 3), when taking the train between Calais and Folkestone. The On Job variable is still set.

**To Reproduce**
1. Use the plugin.
2. Take a job.
3. Take the train between Calais - Folkestone.
4. All Job Info is lost.

**Expected behavior**
Job info shouldn't be lost. 😉 

**Screenshots**
**1.**
![SDKVersion](https://user-images.githubusercontent.com/32167597/126033247-dfe068eb-1646-4add-9e03-8275511356b7.PNG)  
**2.**
![SDKJobBeforeTrain](https://user-images.githubusercontent.com/32167597/126033245-d03338ba-82c0-4d5e-8b13-1f3fea1d61f1.PNG)  
**3.**
![SDKAfterTrain](https://user-images.githubusercontent.com/32167597/126033244-51e6a193-f292-4afb-8981-f55f7dba55d6.PNG)  

**Desktop (please complete the following information):**
 - OS: W10
 - Browser: N/A
 - Version 1.16

**Additional context**
I have made sure the plugin was installed correctly. I have tried the plugin versions 1.12 and 1.11

**Comment by Pnaop** [comment]
I noticed that in the changelog, Version 1.10 Rev 7 Includes a fix for the train: [link](https://github.com/RenCloud/scs-sdk-plugin/blob/master/changelog.md)

The newest release is version 1.10.6 though. Where can I get the newest release? Or should I build that myself?

**Comment by RenCloud** [comment]
Yes, that is correct. The bug was fixed, but no release build was created. So currently you need to build it yourself.
